apps/send/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "send",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "devDependencies": {
```

Comment:
I'm not sure all the dependencies listed here should be `devDependencies`. Some are, some are not IMHO